tests/aggregation_regress/tests.py

```diff
@@ -1914,7 +1914,11 @@ def test_aggregate_group_by_unseen_columns_unmanaged(self):
 
 
 class JoinPromotionTests(TestCase):
-    def test_ticket_21150(self):
+    def test_select_related_with_annotation(self):
+        """
+        Select_related with annotation returns the correct results
+        and maintains related objects (#21150).
+        """
```

Comment:
true, but I think we should keep it to:
* Do not lose the issue number.
* Do not have a docstring for just the issue number.

Reply:
Yes, but we still have issue numbers in commit messages and this is usually how we track the issue number rather than adding them into doc strings.
I don't mind too much if we already have a doc string and we had the number in the test name, but otherwise, the doc string needs to add value beyond a ticket number

Reply:
I could loop over all docstring and remove the issue numbers, 
but then we'll have to change the coding style mentioned in the docs? 

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style

I don't think every character has to add value per say, but keeping more tests consistent with the style, will put pressure on whoever is writing future tests for follow it ?